src/main/kotlin/com/exactpro/th2/check1/rule/AbstractCheckTask.kt

```diff
@@ -337,6 +377,42 @@ abstract class AbstractCheckTask(
         }
     }
 
+    private fun doAfterCompleteEvent() {
+        if (untrusted && !isIgnoreUntrustedFlag()) {
```

Comment:
Why do we still have `isIgnoreUntrustedFlag()` method? I don't understand why do we need it. Could you please explain this to me one more time?